tests/test_packaging.py

```diff
@@ -45,12 +43,7 @@ def __str__(self) -> str:
 
 
 def test_check_pypi_rendering():
-    subprocess.check_call(["python3", "setup.py", "sdist"])
```

Comment:
Removed this line as well to fix the test. The sdist build is essentially tested with each CI test run (`pip install .[test]`) as well as with the dedicated "Build & Publish" job for changes to `setup.py` and `pyproject.toml`.